doc/nrf/dm_ncs_distro.rst

```diff
@@ -0,0 +1,86 @@
+.. _dm_ncs_distro:
+
+Redistributing NCS
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section is intended for users that would like to re-distribute the |NCS|
+with additions or changes to it.  This can be useful if you are a software or
+hardware vendor that would like to augment the |NCS| with additional
+functionality, while at the same time still benefiting from the infrastructure
+it provides.
+
+The contents of this section are not comprehensive, they are intended as
+suggestions for those wanting to extend the |NCS| but other mechanisms may be
+used as long as they comply with the text in the :file:`LICENSE` file that you
+can find at the root of every Git repository in the |NCS|. In case of doubt
+please reach out to Nordic Semiconductor.
+
+Getting added to the main manifest
+**********************************
+
+The simplest way (for your users or customers) to become part of the |NCS| is to
+be included in its official `west manifest file`_. In this way your own
+:ref:`module repository <zephyr:modules>` will automatically be part of the
+|NCS| official distribution and will be included in all releases.
+
+This solution is reserved for selected vendors and requires a commercial
+agreement with Nordic Semiconductor. Please contact your local representative if
+you'd like to be considered for inclusion.
+
+Some examples of this type of integration are:
+
+* :ref:`Memfault <mod_memfault>`. This functionality is present via the addition
+  of a `west project in the |NCS| manifest <Memfault entry in the manifest>`_.
```

Comment:
@peknis this type of link doesn't seem to work with Sphinx. Can you suggest an alternative?

Reply:
You were just missing the underscore as fixed in my suggestion.

Reply:
It should be \`west project in the nRF Connect SDK manifest \<Memfault entry in the manifest_\>\`_. I don't think the shortcut will work here.

Reply:
Actually it does! I checked and @peknis is right, just adding the underscore fixes i!

Reply:
Funnily... it works sometimes. This particular one works fine, but others don't seem to. Very strange.

Reply:
Actually scratch that. It seems to work fine now.